frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx

```diff
@@ -50,6 +50,7 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
     groupPermissionService.getGroup(groupPermissionId).then((data) => {
       this.setState({
         groupPermission: data,
+        currentTab: data?.group !== 'admin' ? 'users' : 'apps',
```

Comment:
looks like we should reverse the condition, should be
```
currentTab: data?.group !== 'admin' ? 'apps' : 'users',
```

Reply:
done